docs/config.json

```diff
@@ -2033,6 +2316,16 @@
     }
   },
   "redirects": [
+    {
```

Comment:
Was this PR intended to add these redirects?

Reply:
I just grabbed the `config.json` that was in place before my commit, and added the new policy entries.